fractal-theme/assets/js/src/search-results.js

```diff
@@ -0,0 +1,75 @@
+import Fuse from 'fuse.js'
+
+function fractalSearchResults() {
+  const searchComponent = document.querySelector('.search');
+  const searchInput = document.getElementById('searchPageInput');
+  const searchResults = document.getElementById('searchResults');
+  const SearchApi = '/assets/theme/js/search.json';
```

Comment:
If `search.json` is being built with gulp couldn't you simply import it as a static asset and avoid `fetch` altogether?

Reply:
I guess we could. I just tried to use best practice using promises. But you are right I will consider to change this next time I am touching search.